src/alarm_trap_sender.cpp

```diff
@@ -241,7 +96,7 @@ void AlarmTrapSender::alarm_trap_send_callback(int op,
 // retires if needed.
 void AlarmTrapSender::send_trap(const AlarmTableDef& alarm_table_def)
 {
-  TRC_WARNING("Trap with alarm ID %d.%d being sent", alarm_table_def.alarm_index(), alarm_table_def.state());
+  TRC_INFO("Trap with alarm ID %d.%d being sent", alarm_table_def.alarm_index(), alarm_table_def.state());
```

Comment:
I kind of liked it as a warning because traps usually mean something's wrong, and it's nice to have a logfile recording what traps went out when.